docs/guidance/related_item_guide.rst

```diff
@@ -33,120 +33,349 @@ The related item that the resource is published in may have an identifier of thi
-      <affiliation affiliationIdentifier="https://ror.org/03efmqc40" affiiationIdentifierScheme="ROR" SchemeURI="https://ror.org">Arizona State University</affiliation>
-    </creator>
-  </creators>
-  <titles>
-    <title xml:lang="en-US">Example Article Title/title>
-  </titles>
-  <publisher xml:lang="en">Example Publisher</publisher>
-  <publicationYear>2022</publicationYear>
-  <resourceType resourceTypeGeneral="JournalArticle"></resourceType>
-  <relatedIdentifiers>
-    <relatedIdentifier relatedIdentifierType="ISSN" relationType="IsPublishedIn">1234-5678</relatedIdentifier>
-  </relatedIdentifiers>
-  <relatedItems>
-    <relatedItem relationType="IsPublishedIn" relatedItemType="Journal">
-      <relatedItemIdentifier relatedItemIdentifierType="ISSN">1234-5678</relatedItemIdentifier>
+.. tab-set-code::
+  
+  .. code:: xml
+
+    <?xml version="1.0" encoding="UTF-8"?>
+    <resource
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns="http://datacite.org/schema/kernel-4" xsi:schemaLocation="http://datacite.org/schema/kernel-4 http://schema.datacite.org/meta/kernel-4.4/metadata.xsd">
+      <identifier identifierType="DOI">10.21384/ExampleArticle</identifier>
+      <creators>
+        <creator>
+          <creatorName nameType="Personal">Garcia, Sofia</creatorName>
+          <givenName>Sofia</givenName>
+          <familyName>Garcia</familyName>
+          <nameIdentifier schemeURI="https://orcid.org/" nameIdentifierScheme="ORCID">0000-0001-5727-2427</nameIdentifier>
+          <affiliation affiliationIdentifier="https://ror.org/03efmqc40" affiiationIdentifierScheme="ROR" SchemeURI="https://ror.org">Arizona State University</affiliation>
+        </creator>
+      </creators>
       <titles>
-        <title>Journal of Metadata Examples</title>
+        <title xml:lang="en">Example Article Title</title>
       </titles>
+      <publisher xml:lang="en">Example Publisher</publisher>
       <publicationYear>2022</publicationYear>
-      <volume>3</volume>
-      <issue>4</issue>
-      <number numberType="Article">2</number>
-      <firstPage>20</firstPage>
-      <lastPage>35</lastPage>
-      <publisher>Example Publisher</publisher>
-    </relatedItem>
-  </relatedItems>
-
+      <resourceType resourceTypeGeneral="JournalArticle"></resourceType>
+      <relatedIdentifiers>
+        <relatedIdentifier relatedIdentifierType="ISSN" relationType="IsPublishedIn">1234-5678</relatedIdentifier>
+      </relatedIdentifiers>
+      <relatedItems>
+        <relatedItem relationType="IsPublishedIn" relatedItemType="Journal">
+          <relatedItemIdentifier relatedItemIdentifierType="ISSN">1234-5678</relatedItemIdentifier>
+          <titles>
+            <title>Journal of Metadata Examples</title>
+          </titles>
+          <publicationYear>2022</publicationYear>
+          <volume>3</volume>
+          <issue>4</issue>
+          <number numberType="Article">2</number>
```

Comment:
Most journals either use an article number or page numbers. Should the example include both or does that make it more confusing?

Reply:
That's a good point! It wouldn't have struck me as confusing, but if it jumps out to folks we can use either an article number or a page number range in this example.

Reply:
I've removed the article number from this example! The other one is a chapter number which I think should be clear.